applications/serial_lte_modem/src/slm_at_commands.c

```diff
@@ -212,7 +197,7 @@ static int handle_at_shutdown(enum at_cmd_type type)
 FUNC_NORETURN void slm_reset(void)
 {
 	slm_at_host_uninit();
-	modem_power_off();
```

Comment:
I don't really know. I didn't change the existing functionality. Maybe we could make some changes to this. Possibly in some cases `AT+CFUN=0` is desired so that possible changes made to the modem configuration are written to NVM? This could be looked into.

Reply:
slm_power_off_modem does not have the check for flight mode, that modem_power_off had. I think that it changes existing functionality.

Reply:
Ah. `LTE_LC_FUNC_MODE_POWER_OFF`, that used to be checked in `modem_power_off()`, is actually `0`. It's not flight mode.
I figured I would just remove it to simplify (and cause confusion it seems) as if sending `AT+CFUN=0` while `AT+CFUN?` is already 0 will just return right away.

Reply:
Ah, so it is. Mixed up LTE_LC_FUNC_MODE_OFFLINE and LTE_LC_FUNC_MODE_POWER_OFF. :P

Reply:
Even though you didn't say it, I agree that the names can be confusing. I feel like this is not the first time this is happening.